src/components/dialogs/addProject.tsx

```
@@ -28,6 +28,7 @@ import {
   Dialog,
   DialogTrigger,
   DialogContent,
+  
   DialogHeader,
```

ESLint

[error] Delete `⏎··`

(prettier/prettier)